libraries/AP_Mission/AP_Mission.h

```diff
@@ -763,6 +763,8 @@ class AP_Mission
 
     bool is_valid_index(const uint16_t index) const { return index < _cmd_total; }
 
+    bool is_resume() const { return _restart == 0; }
```

Comment:
Let's add a comment above this new method, "returns true if configured to resume"

I also think we should move this up to be closer to other related methods.  I think directly below "set_force_resume()" might be good.

Also it makes no difference to Rover I think the return should also check "_force_resume"

Reply:
@rmackay9 Thank you for your comment. I fixed it.